controller/UserController.js

```diff
@@ -0,0 +1,21 @@
+import passwordHash from 'password-hash';
+
+
+class UserController {
+    signUp(req, res) {
```

Comment:
User signup is not creating the user here.

You just read the request params and did nothing with it

You have to create the user record here nad return a response following the specification in the ticket